docs/index.html

```diff
@@ -1,234 +1,141 @@
-<!DOCTYPE html>
-
-<html lang="en">
+<!doctype html>
```

Comment:
@mehalter, can you please exclude `/docs` from the github source code and use a GitHub action to generate them on merge? Including them in the source creates **a lot** of noise and adds additional friction to the dev process.

Reply:
Fixed this in latest commit. Removed the `/docs` folder and added a github action to generate docs on pushes to master to the `gh-pages` branch.